plot/plot.py

```diff
@@ -156,7 +162,7 @@ def save_split_vln_plot(df, out_path, ymax):
 def generate(in_file, out_file, ceiling = None):
     df = parse_data(in_file)
 
-    if ceiling is None:
```

Comment:
`if ceiling is None` is the preferred way to check for `None`. See https://peps.python.org/pep-0008/#programming-recommendations

Reply:
ceiling was 0 on my machine, not none. I didn’t investigate why. Would you like me to?

this handles both 0 and none

Reply:
ah, it looks like the default value is set to 0 on line 35. Can we set it to `None` there?